README.md

```diff
@@ -1,39 +1,37 @@
-# FHIR Information Gateway
+# FHIR Info Gateway
 
 <!-- Build status of the main branch -->
 
 [![Build Status](https://storage.googleapis.com/fhir-proxy-build-badges/build.svg)](https://storage.googleapis.com/fhir-proxy-build-badges/build.html)
 
-FHIR Information Gateway is a simple access-control proxy that sits in front of
-a [FHIR](https://www.hl7.org/fhir/) store (e.g., a
+The FHIR Info Gateway is an access-control proxy that sits in front of a
+[FHIR](https://www.hl7.org/fhir/) store (e.g., a
 [HAPI FHIR](https://hapifhir.io/) server,
 [GCP FHIR store](https://cloud.google.com/healthcare-api/docs/concepts/fhir),
 etc.) and controls access to FHIR resources.
 
-Note: "gateway" and "proxy" are used interchangably here, as the gateway is
-implemented as a proxy server.
-
 The authorization and access-control have three components; one of them is this
-access proxy. The other two are an Identity Provider (IDP) and an Authorization
+FHIR Info Gateway. The other two are an Identity Provider (IDP) and an Authorization
 server (AuthZ). The responsibility of this pair is to authenticate the user and
 issue access tokens (in JWT format and using authorization flow of OAuth 2.0).
-The requests to the access proxy should have the access token as a Bearer
+The requests to the FHIR Info Gateway should have the access token as a Bearer
 Authorization header. Based on that, the proxy decides whether to grant access
 for a FHIR query.
 
-<img src="doc/summary.png" width=50% alt="Modules involved in FHIR authorization/access-control">
+![Modules involved in FHIR authorization/access-control](doc/architecture.png)
```

Comment:
nit: consider keeping the smaller image size that was in the old version; also can we remove "Info" from the image, let's just go with "FHIR Gateway" in the image.